ledger/accountdb.go

```diff
@@ -1075,7 +1298,56 @@ func accountsCreateResourceTable(ctx context.Context, tx *sql.Tx) error {
 	return nil
 }
 
-type baseOnlineAccountData struct {
+func accountsCreateOnlineAccountsTable(ctx context.Context, tx *sql.Tx) error {
+	var exists bool
+	err := tx.QueryRowContext(ctx, "SELECT 1 FROM pragma_table_info('onlineaccounts') WHERE name='address'").Scan(&exists)
+	if err == nil {
+		// Already exists.
+		return nil
```

Comment:
Why do we check for existence, since the `createOnlineAccountsTable` queries use `if not exist` to protect from trying to recreate the table (or index)?